content/participants/ttjc@panchami6/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: panchami6
+type: "participant"
+name: "Panchami Thulupule"
+portfolio: ""
+github-link: "https://github.com/panchami6"
+linkedin-link: "https://www.linkedin.com/in/panchami-thulupule-889583170/"
+looking-for: "job"
+description: "Python | Java | Learning JavaScript"
+---
+
+Hi! I am Panchami Thulupule, let me walk you through my intro.
+
+### introduction
+
+Currently working as a Test Engineer.
+
+## why are you taking this challenge?
+
+I need a job.
+I want to learn new technologies and implement them.
+
+## something interesting about you?
+
+I sing and dance.
+
+### help I need?
+
+I am practicing HTML, CSS and Learning JavaScript. I would like to have some project ideas where i can implement them.
+
+### how to contact me?
+
+- LinkedIn
```

Comment:
Don't you have any LinkedIn account add it or remove this,
Suggestions create one you will need it #required

Reply:
I just added the link to my LinkedIn account. 
Thank You.